source/server/GameServer.py

```diff
@@ -184,15 +187,13 @@ def Send_publicInfo(self):
 
         #NOTE: visible_cards needs to be serialized form to be transmitted.
         # On server keep them in serialized form.
-
         if self.rules.Shared_Board:
             # Shared_Board is True: (e.g. Liverpool) -- each player transmits entire board of visible_cards to server.
-            self.Send_broadcast({"action": "publicInfo", "player_names": [p.name for p in self.players],"visible_cards": [self.visible_cards_now],"hand_status": [p.hand_status for p in self.players]})
+            self.Send_broadcast({"action": "publicInfo", "player_names": [p.name for p in self.players],"visible_cards": [self.visible_cards_now],"hand_status": [p.hand_status for p in self.players], "ruleset": self.ruleset})
```

Comment:
is the ruleset actually USED from the broadcast? THere is an explicit call to request it, indpendent of this.

Reply:
Removed it from broadcast, and ran program to insure it wasn't used somewhere non-obvious.